Safety/AttitudeManager/Inc/AttitudeDatatypes.hpp

```diff
@@ -6,25 +6,36 @@
 // Output of the PID module and input to the OutputMixing module
 typedef struct
 {
-    float rollPercent;
-    float pitchPercent;
-    float rudderPercent;
-    float throttlePercent;
+    float frontLeftMotorPercent;
+    float frontRightMotorPercent;
+    float backLeftMotorPercent;
+    float backRightMotorPercent;
```

Comment:
Renamed struct variables to those matching the quadcopter motors instead of control surfaces in a fixed-wing